service/resource/certificate/error.go

```diff
@@ -10,3 +10,10 @@ var invalidConfigError = microerror.New("invalid config")
 func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
+
+var missingCertificateError = microerror.New("missing certificate")
```

Comment:
I think `certificate.IsMissingCertificate` reads nicer than `certificate.IsMissing`, but not certain.

Reply:
`certificate.IsMissingCertificate` stutters. More go idomatic is `certificate.IsMissing`.

Reply:
Done